src/core_plugins/console/public/src/sense_editor/mode/script_highlight_rules.js

```diff
@@ -0,0 +1,66 @@
+
+let ace = require('ace');
+let oop = ace.require("ace/lib/oop");
+let TextHighlightRules = ace.require("ace/mode/text_highlight_rules").TextHighlightRules;
+
+let painlessKeywords = (
+  "def|int|long|byte|String|new|null|for|if|return|ctx"
+);
+
+var ScriptHighlightRules = function () {
```

Comment:
I have the same suggestion here about using native class syntax as I did in the previous file.